pkg/node/address.go

```diff
@@ -284,6 +293,11 @@ func GetIPv4() net.IP {
 	return clone(n.GetNodeIP(false))
 }
 
+func GetAddrV4() *netip.Addr {
+	n := localNode.Get()
+	return cloneAddr(n.GetNodeAddr(false))
+}
```

Comment:
Hm, I don't think it makes much sense to duplicate entire functions just to convert one package (BGP) over to the `netip.Addr` type. I think we'd prefer the entire `pkg/node` to be converted and then downstream packages from there.

I think this approach introduces unnecessary code churn in the tree without an immediate benefit, at the cost of the duplicated functions having bug(s) / regression(s).

Reply:
> Hm, I don't think it makes much sense to duplicate entire functions just to convert one package (BGP) over to the netip.Addr type. I think we'd prefer the entire pkg/node to be converted and then downstream packages from there.

I originally took this approach, see #25485 for details. However, changing the node pkg turned into a massive PR.